### DISCUSSION TITLE
Chip Side - BLE Strength

### DISCUSSION THREAD

**Discussion by twister36** [ESPresense/ESPresense#966] [General]
So I've noticed that how the ESP32 device is pointed into a room makes a difference (one without an external antenna). Has anyone else noticed that devices on directly in front of the ESP32's chip side appear to be closer than the devices on the non-chip side. Now this makes sense, but I just want to confirmed my results for proper mounting.

**Comment by DTTerastar** [comment]
Yes, I believe that is the case for the devices with a circuit board antenna.
